core/deployment/src/main/resources/dev-ui/qwc-chat.js

```diff
@@ -96,6 +121,15 @@ export class QwcChat extends LitElement {
             <div><vaadin-checkbox checked label="Enable Retrieval Augmented Generation (if a RetrievalAugmentor bean exists)"
                                   @change="${(event) => {
                                       this._ragEnabled = event.target.checked;
+                                      this._streamingChatEnabled = this._streamingChatEnabled && !this._ragEnabled;
+                                      this.render();
+                                  }}"/></div>
+            <div><vaadin-checkbox label="Enable Streaming Chat (if supported by model & rag disabled)"
+                                  class="${this.jsonRpc.isStreamingChatSupported() ? 'show' : 'hide'}"
```

Comment:
This jsonrpc method should be called only once - this way you call it again every time the form is rendered